google/cloud/spanner/samples/samples.cc

```diff
@@ -527,6 +527,126 @@ void FieldAccessOnNestedStruct(google::cloud::spanner::Client client) {
 }
 //! [END spanner_field_access_on_nested_struct]
 
+void Send(std::string) {}  // NOLINT
```

Comment:
Maybe accept by const-ref here instead of the NOLINT?

Reply:
+1